src/ImJoyPluginAPI.js

```diff
@@ -253,6 +253,14 @@ class ImJoyPluginAPI {
     return this.viewer.getImagePiecewiseFunctionGaussians(component, name)
```

Comment:
@PaulHax These are no longer valid? They should be removed?

Reply:
Yes.  I just removed them.